src/VscPreviewExtension/src/contentProvider.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { workspace, ExtensionContext, TextDocumentContentProvider, EventEmitter, Event, Uri } from "vscode";
+import * as path from "path";
+
+export class ContentProvider implements TextDocumentContentProvider {
+    private _context: ExtensionContext;
+    private _onDidChange = new EventEmitter<Uri>();
+    protected _content: string;
+    public port;
```

Comment:
I don't find the recommended order of this modifier in the ts doc , so I think we should be consistent with C#

---
In reply to: [80485982](https://github.com/dotnet/docfx/pull/716#discussion_r80485982) [](ancestors = 80485982)